topostats/plotting.py

```diff
@@ -24,44 +23,6 @@
 # pylint: disable=too-many-locals
 
 
-def create_parser() -> arg.ArgumentParser:
```

Comment:
Would need this reinstating too for `toposum` entry point to work.

Reply:
Ignore me, I've no looked at `entry_point.py` and see you've moved all of these there.